helm/netobserv/templates/flows.netobserv.io_flowcollectors.yaml

```
@@ -0,0 +1,8394 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.14.0
+  creationTimestamp: null
+  name: flowcollectors.flows.netobserv.io
+spec:
+  conversion:
+    strategy: Webhook
+    webhook:
+      clientConfig:
+        service:
+          name: netobserv-webhook-service
+          namespace: netobserv

```

We surely want the namespace to be configurable: 

```suggestion
          namespace: {{ .Release.Namespace }}
```